packages/semantic-ui/src/FieldTemplate/FieldTemplate.js

```diff
@@ -15,28 +16,43 @@ function FieldTemplate({
   label,
   rawErrors = [],
   rawHelp,
+  hidden,
   rawDescription,
   ...props
 }) {
   const semanticProps = getSemanticProps(props);
   const { wrapLabel, wrapContent } = semanticProps;
   const errorOptions = getSemanticErrorProps(props);
-  return (
-    <Form.Group key={id} widths="equal" grouped>
-      <MaybeWrap wrap={wrapContent} className="sui-field-content">
-        {children}
-        {displayLabel && rawDescription && (
-          <MaybeWrap wrap={wrapLabel} className="sui-field-label">
-            {rawDescription && (
-              <DescriptionField description={rawDescription} />
-            )}
-          </MaybeWrap>
-        )}
-        <HelpField helpText={rawHelp} id={id + "__help"} />
-        <RawErrors errors={rawErrors} options={errorOptions} />
-      </MaybeWrap>
-    </Form.Group>
-  );
+  
+  if (hidden) {
+    return children;
+  }
+
+  return (<WrapIfAdditional
```

Comment:
Did you mean to change the pattern from:
```
return (
 <xxx>
   <yyy />
 </xxx>
);
```
to
```
return (<xxx>
  <yyy/>
</xxx>);
```
???

Reply:
not sure, will remove, saw it on antd implementation.